src/app/model/jit/operations-event.ts

```diff
@@ -30,5 +31,6 @@ export interface OperationsEvent {
   timestampDefinitionID?: string;
   facilityCode: string;
   vesselPosition?: VesselPosition;
+  portCallPhaseTypeCode: PortCallPhaseTypeCode
```

Comment:
This can be null - aren't you missing a "?" at the end of the variable name?

Reply:
As I understand it, `?` means the field is sometimes absent (`undefined`) rather than `null`, so this might be right.

ICBW though as it is not my strong suit.

Reply:
I am basing my arguments on https://www.educba.com/typescript-nullable/